Parsers/SentinelOne/SentinelOne.txt

```diff
@@ -0,0 +1,277 @@
+                DataByUser,
+                DataRole,
+                DataUserScope,
+                EventTypeDetailed,
+                DataSource,
+                DataExpiryDateStr,
+                DataExpiryTime,
+                DataNetworkquarantine,
+                DataRuleCreationTime,
+                DataRuleDescription,
+                DataRuleExpirationMode,
+                DataRuleId,
+                DataRuleName,
+                DataRuleQueryDetails,
+                DataRuleQueryType,
+                DataRuleSeverity,
+                DataScopeId,
+                DataStatus,
+                DataSystemUser,
+                DataTreatasthreat,
+                DataUserId,
+                DataUserName,
+                EventSubStatus,
+                AgentId,
+                DataComputerName,
+                DataExternalIp,
+                DataGroupName,
+                DataSystem,
+                DataUuid,
+                GroupId,
+                GroupName,
+                DataGroup,
+                DataOptionalGroups,
+                DataCreatedAt,
+                DataDownloadUrl,
+                DataFilePath,
+                DataFilename,
+                DataUploadedFilename,
+                Comments,
+                DataNewValue,
+                DataPolicyId,
+                DataPolicyName,
+                DataNewValueb,
+                DataShouldReboot,
+                DataRoleName,
+                DataScopeLevelName,
+                ActiveDirectoryComputerDistinguishedName,
+                ActiveDirectoryComputerMemberOf,
+                ActiveDirectoryLastUserDistinguishedName,
+                ActiveDirectoryLastUserMemberOf,
+                ActiveThreats,
+                AgentVersion,
+                AllowRemoteShell,
+                AppsVulnerabilityStatus,
+                ComputerName,
+                ConsoleMigrationStatus,
+                CoreCount,
+                CpuCount,
+                CpuId,
+                Domain,
+                EncryptedApplications,
+                ExternalId,
+                ExternalIp,
+                FirewallEnabled,
+                GroupIp,
+                InRemoteShellSession,
+                Infected,
+                InstallerType,
+                IsActive,
+                IsDecommissioned,
+                IsPendingUninstall,
+                IsUninstalled,
+                IsUpToDate,
+                LastActiveDate,
+                LastIpToMgmt,
+                LastLoggedInUserName,
+                LicenseKey,
+                LocationEnabled,
+                LocationType,
+                Locations,
+                MachineType,
+                MitigationMode,
+                MitigationModeSuspicious,
+                ModelName,
+                NetworkInterfaces,
+                NetworkQuarantineEnabled,
+                NetworkStatus,
+                OperationalState,
+                OsArch,
+                DvcOs,
```

Comment:
Should it not be 'SrcDvcOs'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Thanks,
Aditi Maheshwari

Reply:
Fixed